### PR TITLE
Feature/nginx role

### LINKED ISSUE #107: Create flexible nginx role with templating

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
Closes #107 
Using nginx galaxy role instead of manual configuring
Simplified scenario4_image_setup

### CODE REVIEW

File: playbooks/scenario1_setup_telegraf_only.yml [hunk 1]
```diff
@@ -3,42 +3,42 @@
...
+        - name: http_listener_v2
+          options:
+            service_address: ":8080"
+            data_format: "influx"
+  - role: geerlingguy.nginx
+    vars:
+      nginx_vhosts:
+        - listen: "443 ssl"
+          server_name: "csm.outcatcher.com"
+          extra_parameters: |
```
Comment (outcatcher) [inline playbooks/scenario1_setup_telegraf_only.yml]: Maybe move `nginx_vhosts` to variable file?

File: playbooks/scenario4_image_setup.yml [hunk 2]
```diff
@@ -18,14 +18,12 @@
...
-    become: yes
-    apt:
-      name: ['nginx', 'nginx-extras']
 
-  - name: start nginx
-    become: yes
-    service:
-      name: nginx
-      state: started
+- name: Start test Server
```
Comment (outcatcher) [inline playbooks/scenario4_image_setup.yml]: Is it needed to start server during image creation?

Comment (anton-sidelnikov) [reply playbooks/scenario4_image_setup.yml]: i think it's a good idea, we can catch a problem, if it occur, with server while image creation. It's better than face this when infrastructure will be already deployed

File: playbooks/scenario1_setup_telegraf_only.yml (hunk 1 above)
Comment (anton-sidelnikov) [reply playbooks/scenario1_setup_telegraf_only.yml]: hmm, ok

File: playbooks/scenario4_image_setup.yml (hunk 2 above)
Comment (outcatcher) [reply playbooks/scenario4_image_setup.yml]: Seems legit

**Pull request merged**